gotham-client/src/utilities/requests.rs

```diff
@@ -15,7 +15,7 @@ pub fn post(client: &reqwest::Client, path: &str) -> Option<String> {
     let start = PreciseTime::now();
 
     let res = client
-        .post(&format!("http://127.0.0.1:8000/{}", path))
+        .post(&format!("http://localhost:8000/{}", path))
```

Comment:
Any reason for this change? We can leave it like this I am just curious

Reply:
Yes.
When I cam to test `integrations-tests` the server was launched on `localhost` (even though I used the same methods for launch) and requests to `127.0.0.1` failed.
Changing all to `localhost` solved it.